### ISSUE TITLE
Weighted Cross Entropy loss function

### ISSUE THREAD

**Issue by Jorisfournel** [eragonruan/refinenet#10]
Hi,
First thank you for this implementation.
I didn't test it yet but I'm about to, for custom data, a 3 classes segmentation task,
where implementing a weighted cross entropy function should be determinant.
What changes in the code should be done to do that ? 
(I don't master tf yet, I worked on caffe before, a tip would be gratefuly taken)

Thanks in advance !

**Comment by eragonruan** [comment]
@Jorisfournel in [nets/model.py](https://github.com/eragonruan/refinenet-image-segmentation/blob/ef77407fb1abc7f72f421d63bc409acebeb2cbea/nets/model.py#L137), you should modify this function. tf provide weighted cross entropy api, you can find it [here](https://www.tensorflow.org/api_docs/python/tf/nn/weighted_cross_entropy_with_logits)
